sdk/face/ai-vision-face-rest/samples-dev/aadAuth.ts

```diff
@@ -0,0 +1,36 @@
+import { readFileSync } from 'fs';
+import { DefaultAzureCredential } from '@azure/identity';
+
+import createFaceClient, {
+    isUnexpected,
+} from '@azure-rest/ai-vision-face';
+
+/**
+ * This sample demonstrates how to create a liveness detection session.
```

Comment:
pls update the example comment for all samples

Reply:
That's a huge miss... sorry 😅